### DISCUSSION TITLE
There is a problem with Typesafe-i18n demo for Sveltekit when updating packages

### DISCUSSION THREAD

**Discussion by AlexanderJinoridze** [ivanhofer/typesafe-i18n#670] [General]
Long story short when I cloned the Typesafe-i18n Sveltekit demo ([https://github.com/ivanhofer/typesafe-i18n-demo-sveltekit](https://github.com/ivanhofer/typesafe-i18n-demo-sveltekit)) and updated packages there with npm update, I noticed one strange thing that I think is related to the LocaleSwitcher.svelte component in the lib directory, I noticed that when switching languages lang attribute in HTML tag stops changing, can you check it out please? I can't really figure out what's going on I think previously before the update LocaleSwitcher.svelte was rerunning script after clicking anchor changing lang attr and some other stuff but after the update it just doesn't mb they they changed some anchor behaviour in the kit

**Comment by ivanhofer** [comment]
Hi, can you please create a repository where the error occurs?
